pkg/cmd/util/output/backup_describer.go

```diff
@@ -83,6 +83,11 @@ func DescribeBackup(
 
 		d.Printf("Phase:\t%s%s\n", phaseString, logsNote)
 
+		if backup.Spec.ResourcePolicy != nil {
+			d.Println()
+			d.Printf("Resource policies:\t%s\n", backup.Spec.ResourcePolicy.Name)
```

Comment:
Can a backup be associated with multiple resource policies ?

Reply:
no, but one resource policies could be associated with multiple backup.

Reply:
This is the name of the configmap right?  
IMO it would be better to also output the reference type.

Reply:
updated